doc/nvim-tree-lua.txt

```diff
@@ -636,6 +638,7 @@ Defaults to:
       { key = "s",                            action = "system_open" },
       { key = "q",                            action = "close" },
       { key = "g?",                           action = "toggle_help" },
+      { key = 'W',                            action = "collapse_all" }
```

Comment:
could you add this line in the readme too :) ?

Reply:
Sorry... I have added it now.